src/wp-includes/class-wp-user-query.php

```diff
@@ -1051,12 +1061,22 @@ public function __get( $name ) {
 	 *
 	 * @param string $name  Property to check if set.
 	 * @param mixed  $value Property value.
-	 * @return mixed Newly-set property.
 	 */
 	public function __set( $name, $value ) {
 		if ( in_array( $name, $this->compat_fields, true ) ) {
-			return $this->$name = $value;
+			$this->$name = $value;
+			return;
 		}
+
+		_doing_it_wrong(
+			__METHOD__,
+			sprintf(
+			// translators: 1: The name of the non-existent class property.
+				__( 'The "%1$s" property isn\'t defined. Dynamic properties are deprecated in PHP 8.2 and above.' ),
```

Comment:
Replace `isn\'t` with `is not`.

Reply:
Fixed in 42ba291e72e740a6cd4121e78089dccd2a578c35